zeq/http/server.cpp

```diff
@@ -310,6 +310,18 @@ const servus::URI& Server::getURI() const
     return _impl->uri.toServusURI();
 }
 
+SocketDescriptor Server::getSocketDescriptor() const
```

Comment:
Having access to the socket handle is very useful in conjunction with a QSocketNotifier. It handles the poll() as part of the QEventLoop and emits an activated() QSignal when the socket has new data available. This signal can be connected to a lamba which calls recieive(), so with a few lines of code you have a server which automatically receives without the need for a separate processing thread. I see no easy alternative to that mechanism.